gradle/publish.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
+task sourceJar(type: Jar) {
+    from sourceSets.main.allSource
+    classifier 'sources'
+}
+
+publishing {
+    publications {
+        bintray(MavenPublication) {
```

Comment:
lost the javadoc jar. (which i personally don't care about too much, so i'm fine with it)